src/components/Members/MembersGrid/MemberModal/MemberModal.js

```diff
@@ -209,22 +235,16 @@ class MemberModal extends Component {
                   <h2 className="name-small">
                     { this.memberFullName() }
                   </h2>
-                  <strong>
-                    {' '}
-                    { this.member.role }
-                    {' '}
-                  </strong>
-                  <div className="member-subtitles" style={{ display: (this.member.status === 'inactive') ? 'block' : 'none' }}>
+                  <strong> { this.member.role } </strong>
+                  <div
+                    className='member-subtitles'
+                    style={{display: (this.member.status === 'inactive') ?
+                    'block' : 'none'}}>
                     <h6>
+                      <div> { this.member.subtitle } </div>
                       <div>
-                        {' '}
-                        { this.member.subtitle }
-                        {' '}
-                      </div>
-                      <div>
-                        {' '}
-                        {`Since ${this.member.class}, ${this.member.semesters} semesters` }
-                        {' '}
+                        {'Since ' + this.member.class + ', ' +
+                        this.member.semesters + ' semesters' }
```

Comment:
Nit: It may be easier to read string literals if there are too many variables in the string
```suggestion
                        {'Since ' + this.member.class + ', ' +
                        this.member.semesters + ' semesters' }
                        {`Since ${this.member.class}, ${this.members.semesters} semesters`}
```